inference-contract/docs/notebooks/templates/token.py

```diff
@@ -0,0 +1,182 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.16.1
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+
+# %% [markdown]
+# # Notebook to Invoke Token Operations
+
+# %% [markdown]
+# ## Configure Token Information
+#
+# This section enables customization of the token. Edit the variables in the section below as necessary.
+# * token_owner : the identity of the token owner
+# * token_class : the name of the class of tokens that was minted
+# * token_name : the name of the specific token that was minted
+# * token_path : the full context path to the token
+# * context_file : the name of the context file where token information is located
+#
+# Note that the notebook assumes that there is a key file for the identity of the form
+#
+# ```bash
+# ${keys}/${identity}_private.pem
+# ```
+#
+
+# %% tags=["parameters"]
```

Comment:
Same comment as with token-issuer, duplication of (slightly re-ordered) fields in two code blocks, one of them auto-generated is highly confusing.  Seems to me relying just on the auto-generated one should be completely sufficient ...

Reply:
can't do it. the parameters field is necessary for papermill to inject the parameters into the file.

Reply:
I've verified that the parameters block must have a superset of the variables that are inherited.